apps/backend/test/unit/deposits/cash-deposit.service.spec.ts

```diff
@@ -1,78 +1,68 @@
 import { Logger } from '@nestjs/common';
-import { Test, TestingModule } from '@nestjs/testing';
+import { Test } from '@nestjs/testing';
 import { getRepositoryToken } from '@nestjs/typeorm';
-import { Repository, UpdateResult } from 'typeorm';
+import { In, Repository } from 'typeorm';
 import * as fs from 'fs';
 import path from 'path';
-import { MatchStatus } from '../../../src/common/const';
-import { ParseArgsTDI, FileTypes, Ministries } from '../../../src/constants';
+import { MatchStatus, MatchStatusAll } from '../../../src/common/const';
+import {
+  ParseArgsTDI,
+  FileTypes,
+  Ministries,
+  PaymentMethodClassification
+} from '../../../src/constants';
 import { CashDepositService } from '../../../src/deposits/cash-deposit.service';
 import { CashDepositEntity } from '../../../src/deposits/entities/cash-deposit.entity';
 import { TDI17Details } from '../../../src/flat-files';
 import { parseTDI } from '../../../src/lambdas/utils/parseTDI';
 import { LocationEntity } from '../../../src/location/entities';
+import { MockData } from '../../mocks/mocks';
 
 describe('CashDepositService', () => {
-  let service: CashDepositService;
-  let repository: Repository<CashDepositEntity>;
-  let cashDepositMock: CashDepositEntity;
-  let locationMock: LocationEntity;
-  let updateResult: UpdateResult;
-
+  let cashDepositService: CashDepositService;
+  let cashDepositRepo: Repository<CashDepositEntity>;
+  let cashDeposits: CashDepositEntity[];
   beforeEach(async () => {
-    const module: TestingModule = await Test.createTestingModule({
+    const moduleRef = await Test.createTestingModule({
       providers: [
         CashDepositService,
-        Logger,
         {
           provide: getRepositoryToken(CashDepositEntity),
           useValue: {
             findOneByOrFail: jest.fn(),
-            delete: jest.fn(),
-            where: jest.fn(),
-            execute: jest.fn(),
-            findOneOrFail: jest.fn(() => Promise.resolve({})),
-            findAll: jest.fn().mockResolvedValue([]),
-            save: jest.fn().mockResolvedValue(cashDepositMock),
-            create: jest.fn().mockReturnValue(cashDepositMock),
-            createQueryBuilder: jest
-              .fn()
-              .mockReturnValue({ cash_deposit_source_file_name: 'test/TDI17' })
+            find: jest.fn(),
+            save: jest.fn()
           }
-        }
+        },
+        { provide: Logger, useValue: {} }
       ]
     }).compile();
 
-    service = module.get<CashDepositService>(CashDepositService);
```

Comment:
I believe in your other PR, you kept the service variable name as `service`. Happy for consistency either way; I did kind of like the idea of naming it just `service`, as if to say "this is the service we're actually testing with this spec"